src/main/java/com/puppycrawl/tools/checkstyle/checks/sizes/LineLengthCheck.java

```diff
@@ -59,7 +67,7 @@
  * <li>
  * Property {@code ignorePattern} - Specify pattern for lines to ignore.
  * Type is {@code java.util.regex.Pattern}.
- * Default value is {@code "^$"}.
+ * Default value is {@code "(package|import) .*"}.
```

Comment:
can't be `^(package|import) .*`, we need to match 
```
              import a.b.c;
```
as well.

Reply:
It will be unfortunate limitation by design.
Without validation of beginning of line there will be bunch of false positives on code like:
` int myimport = 0:`

Reply:
There should be beginning of line or space before `import`

Reply:
to not complicate regexp, I would simply use demand for begniing of line. Imports that does not start from the begging of  line is weird code and user should look at it, event it is a false positive :). They can enhance regexp if that is normal in their codebase.

Reply:
Done.